src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+@use './variables' as *;
+
+.product {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 0 16px;
+
+  &__main-image {
+    margin: 32px auto 40px;
+  }
+
+  &__item--name {
+    margin-bottom: 4px;
+  }
+
+  &__item--description {
+    margin-bottom: $margin-main;
+  }
+
+  &__wrapper {
+    margin-bottom: $margin-main;
+  }
+
+  &__button {
+    margin-bottom: $margin-main;
+  }
+}
+
+.main-image {
+  background-image: url(../images/imac.jpeg);
+  width: 160px;
+  height: 134px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: 100% 100%;
+}
```

Comment:
The '.main-image' class does not follow the BEM naming convention. If it is an element of '.product', it should be named '.product__main-image'. If it is a standalone block, then it should remain as '.main-image', but the styling should not be nested under '.product'.